packages/verification-vue-component/src/components/Verification/items/cards/FaqCard.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+    <BaseCard class="faq-card">
+        <template #header>
+            <div class="title">
+                <span class="help-card-title">{{ _$t('verification.card.faq.title') }}</span>
+            </div>
+        </template>
+        <template #body>
+            <div class="faq-items accordion">
+                <div class="item" v-for="(item, index) in _$t('verification.card.faq.items', { returnObjects: true })" :key="index" :class="{ open: isItemOpen(index) }">
+                    <div class="item-title" @click="toggleItem(index)">
+                        <span>{{ item.title }}</span>
+                        <MDIcon :icon="mdiChevronUp" class="chevron"/>
```

Comment:
I think this icon should be gray instead of black, at least it looks like that in the layout.

Reply:
You are right, adjusted it and fixed chevron direction ^^